CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 ## 0.1.1 (Upcoming)
 
+### Features
+
+* commonsteps/http_config: Add `http_content` configuration option as an
+    alternative method for serving static HTTP content. This option works
+    similar to `http_directory` but has the ability to serve files that include
+    Go templating variables that can be interpolated at runtime by Packer core.
+    [[GH-43]()https://github.com/hashicorp/packer-plugin-sdk/pull/43]
+
```

Comment:
Nice ! 
```suggestion
* commonsteps/http_config: Add `http_content` configuration option as an
    alternative method for serving static HTTP content. This option works
    similar to `http_directory` but has the ability to serve files that include
    Go templating variables that can be interpolated at runtime by Packer core.
    [GH-43](https://github.com/hashicorp/packer-plugin-sdk/pull/43)

```

Also, #43 adds a `didyoumean` Package that could allow plugin maintainers to hint on things. I think it would be nice to document it too.

Reply:
Oops to late :p